test/browser/select.test.js

```diff
@@ -28,4 +28,34 @@ describe('Select', () => {
 		render(<App />, scratch);
 		expect(scratch.firstChild.value).to.equal('B');
 	});
+
+	it('should set value with selected', () => {
+		function App() {
+			return (
+				<select>
+					<option value="A">A</option>
+					<option selected value="B">B</option>
+					<option value="C">C</option>
+				</select>
+			);
+		}
+
+		render(<App />, scratch);
+		expect(scratch.firstChild.value).to.equal('B');
+	});
+
+	it('should work with multiple selected', () => {
+		function App() {
+			return (
+				<select multiple>
+					<option value="A">A</option>
+					<option selected value="B">B</option>
+					<option selected value="C">C</option>
+				</select>
+			);
+		}
+
+		render(<App />, scratch);
+		expect(scratch.firstChild.value).to.equal('B');
```

Comment:
Same here: Let's check that both values are selected :)

Reply:
Can we check this with secondChild or? Couldn't figure this out while testing

Reply:
These are standard DOM methods. The DOM doesn't have a `secondChild` property but you can turn all children (= `dom.childNodes`) into an array:

https://stackoverflow.com/a/3199627/755391